opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java

```diff
@@ -351,7 +353,10 @@ private void onFeatureSelectedByNormalClick(Feature feature) {
             listTaskInteractor.fetchFamilyDetails(selectedFeature.id());
         } else if (IRS_VERIFICATION.equals(code) && isZambiaIRSLite()) {
             listTaskInteractor.fetchInterventionDetails(IRS, feature.id(), false);
-        } else if (IRS_VERIFICATION.equals(code) && COMPLETE.equals(businessStatus)) {
+        }else if (CDD_SUPERVISION.equals(code) && isKenyaMDALite() && (NOT_VISITED.equals(businessStatus) || IN_PROGRESS.equals(businessStatus))){
+            startForm(selectedFeature, null, selectedFeatureInterventionType);
```

Comment:
Could we use `listTaskInteractor.fetchInterventionDetails` so that validation of user location is done when `validateFarStructures` is set to true

Reply:
updating this...

Reply:
updated